rest/class.wp-super-cache-rest-get-notices.php

```diff
@@ -28,6 +29,40 @@ public function callback( $request ) {
 	/**
 	 * @param array $notices
 	 */
+	protected function add_preload_notices( & $notices ) {
+		global $cache_enabled, $super_cache_enabled;
+		if ( false == $cache_enabled ) {
+			$notices[ 'preload_disabled_cache_off' ] = array(
+				'type' => 'warning',
+				'message' => __(
+					'Preloading is disabled as caching is disabled',
+					'wp-super-cache'
+				),
+			);
+		}
+		if ( false == $super_cache_enabled ) {
+			$notices[ 'preload_disabled_supercache_off' ] = array(
+				'type' => 'warning',
+				'message' => __(
+					'Preloading is disabled as supercaching is disabled',
+					'wp-super-cache'
+				),
+			);
+		}
+		if ( false === defined( 'DISABLESUPERCACHEPRELOADING' ) ) {
```

Comment:
@donnchawp Should this be checking for `true === defined( 'DISABLESUPERCACHEPRELOADING' )`?